editor/block-settings-menu/unknown-converter.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { IconButton } from '@wordpress/components';
+import { getUnknownTypeHandlerName, rawHandler } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import { getBlock } from '../selectors';
+import { replaceBlocks } from '../actions';
+
+export function UnknownConverter( { block, convertToBlocks } ) {
```

Comment:
I don't have any bright ideas, but I think we should call this differently :)

Reply:
Maybe LegacyConverter or ClassicConverter.

Reply:
I've named it this way because we're checking `getUnknownTypeHandlerName`. :)

Reply:
I mean, this is an API thing, not a block thing. The component shouldn't depend on what block handles unknown pieces right?